src/views/Login.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="container">
+    <img src="../assets/wolox-logo.png" class="logo">
+    <div class="input-container">
+      <label for="name" class="label">First name</label>
+      <input id="name" class="input" v-model="firstName">
+    </div>
+    <div class="input-container">
+      <label for="name" class="label">Last name</label>
+      <input id="name" class="input" v-model="lastName">
+    </div>
+    <div class="input-container">
+      <label for="name" class="label">Email</label>
+      <input id="name" class="input" v-model="email">
+    </div>
+    <div class="input-container">
+      <label for="name" class="label">Password</label>
+      <input id="name" class="input" v-model="password" type="password">
+    </div>
+    <button class="primaryButton" @click="printForm">Sign up</button>
+    <div class="horizontal-division"/>
+    <a :class="'secondaryButton'">Login</a>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "home",
+  components: {},
+  data() {
+    return {
+      firstName: null,
+      lastName: null,
+      email: null,
+      password: null
+    };
+  },
+  methods: {
+    generateObject() {
+      const formObject = {
+        user: {
+          first_name: this.firstName,
+          last_name: this.lastName,
+          email: this.email,
+          password: this.password
+        }
+      };
+      return formObject;
+    },
+    printForm() {
+      console.log(this.generateObject());
+    }
+  }
+};
+</script>
+
+<style lang="scss" scoped>
+@import "../scss/commons/colors.scss";
+
+.container {
+  background-color: $light-grey;
```

Comment:
some props are releated... maybe you can create a generic class or mixin and use a extend